webapp/src/main/java/org/opentestsystem/rdw/admin/model/Group.java

```diff
@@ -44,16 +53,30 @@ public int getStudentCount() {
         return studentCount;
     }
 
+    /**
+     * @return true if this group is deleted
+     */
+    public boolean getIsDeleted() {
+        return isDeleted;
+    }
+
     public static Builder builder() {
         return new Builder();
     }
 
     public static class Builder {
+        private int id;
         private String name;
         private String schoolName;
         private Subject subject;
         private int schoolYear;
         private int studentCount;
+        private boolean isDeleted;
```

Comment:
not sure there is a convention for builders but i would make this `deleted` with the method `deleted` as well